docs/quickstart.md

```diff
@@ -40,72 +42,278 @@
     Use "dce [command] --help" for more information about a command.
     ```
 
-4. Type `dce init` to configure dce via an interactive prompt. This will generate a config file at ~/.dce.yaml
+1. Type `dce init` to generate a new config file at ~/.dce.yaml. Leave everything blank for now.
+
+## Deploying DCE
+
+1. [Download and install the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html)
+
+1. Choose an AWS account to be your new "DCE Master Account" and configure the AWS CLI with user credentials that have AdministratorAccess in that account.
+
+    ```
+    ➜  ~ aws configure set aws_access_key_id default_access_key
+    ➜  ~ aws configure set aws_secret_access_key default_secret_key
+    ```
+
+1. Type `dce system deploy` to deploy dce to the AWS account specified in the previous step.
+
+1. Edit your dce config file with the host and base url from the api gateway that was just deployed to your master account. This can be found in the master account under `API Gateway > (The API with "dce" in the title) > Stages > "Invoke URL: https://<host>/<baseurl>"`. Your config file should look something like this:
+
+    ```
+    api:
+      host: abcdefghij.execute-api.us-east-1.amazonaws.com
+      basepath: /api
+    region: us-east-1
+    ```
+   
+## Authenticating with the DCE System
```

Comment:
Probably worth pointing out here that Cognito is _an option_ for authenticating against DCE.
You can also use straight up AWS creds (shared creds file, env vars, ec2 metadata, etc), and DCE will treat you as an admin user.

Reply:
Just thinking that in my mind "Quick Start" is how some dev is setting this up on their own AWS account, to test it out. They don't really need to do anything with Cognito in that case. 

Cognito auth is a more advanced use case, where you're actually federating in with your enterprise's IDP.

Reply:
I don’t disagree, but I do like having the entire “happy path” documented in detail like this. 

What do you think about this moving to the main repo and becoming our primary “how to dce” doc? Steps with multiple variations could link to separate documents.

Reply:
As per our offline conversation: https://github.com/Optum/dce/issues/159